Tools/openai_manager.py

```diff
@@ -1,11 +1,13 @@
-from typing import Optional, List, Dict, Any
-import base64
+
 from Tools.openai_tools import OpenAIClient
 from config import Config
 from Tools.conversation_manager import ConversationManager
 from Tools.image_helper import encode_image_to_data_url
-from SqlDB.user_cache import UserCache
 from Constants.prompts import CHAT_MAIN_PROMPT
+from SqlDB.user_details_service import is_user_details_exists
+from SqlDB.database import get_db
```

Comment:
database logic should be only used in database medule -> user_details_service. Having it in openai_manager will make code messy.